docs/downstream-builds.md

```diff
@@ -0,0 +1,68 @@
+# Specific instructions for Downstream builds
+
+## How to manually build custom images for OpenShift
+
+1. Set the image urls for your test repos, i.e
+```console
+$ export IMG=quay.io/<username>/instaslice-operator:test
+$ export IMG_DMST=quay.io/<username>/instaslice-daemonset:test
+$ export BUNDLE_IMAGE=quay.io/<username>/instaslice-bundle:test
+```
+1. Build operator and daemonset using OpenShift specifig containerfiles
+```console
+$ make docker-build-ocp
+```
+1. Push images to you test repo
+```console
+$ make docker-push
+```
+1. Build bundle using OpenShift containerfile
+```console
+$ make bundle-build-ocp
+```
+1. Push bundle to test repo
+``` console
+$ make bundle-push
+```
+## Test the images on an OpenShift cluster using emulator mode
+
+### Prerequisites
+- You have a cluster available
+- You have the RH Cert Manager operator installed through OperatorHub
+- You have operator-sdk installed
```

Comment:
It's good to add the references/links to the above pre-reqs for the user to refer/install

Reply:
will do